core/config/config.go

```diff
@@ -164,6 +180,26 @@ var (
 		Required: true,
 		Usage:    "Load configuration from `FILE`",
 	}
+	EnvironmentFlag = cli.StringFlag{
+		Name:     "environment",
+		Required: true,
+		Usage:    "Set the environment (production or development)",
+	}
+	EthRpcUrlFlag = cli.StringFlag{
+		Name:     "eth-rpc-url",
+		Required: true,
+		Usage:    "Ethereum RPC URL",
+	}
+	EthWsUrlFlag = cli.StringFlag{
+		Name:     "eth-ws-url",
+		Required: true,
+		Usage:    "Ethereum WS URL",
```

Comment:
Can you add EnvVar flags to these? See how eigenDA does it: https://github.com/Layr-Labs/eigenda/blob/e7ec7fa7f8dddda8fd64ec77435694d1c5dc9851/disperser/cmd/batcher/flags/flags.go#L25